CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Next
 
+* [#823](https://github.com/DestinyItemManager/DIM/issues/823) Added 'current' property to stores.
```

Comment:
It was one of those updates that seemed useful enough to want to elevate the awareness.